tests/test_assets/model_details/output/model_details_transform_empty_measurements_output.json

```diff
@@ -8,7 +8,7 @@
     "contributing_group": "UCI",
     "study_synid": "syn22964685",
     "rrid": "MMRRC_034830-JAX",
-    "jax_id": 4807,
+    "jax_id": "004807",
```

Comment:
Do you have any tests where an input _is_ 6 digits long, just to test that more 0's don't get added?